tests/test_lfp_amplitude_function.py

```diff
@@ -1,72 +1,72 @@
 from cgi import test
 from turtle import shape
 import numpy as np
-import tklfp
+import src
```

Comment:
this is not right...you shouldn't be importing src. You should be importing wslfp if you set up and installed the package right